src/script/auth/page/Root.tsx

```diff
@@ -73,6 +74,13 @@ const RootComponent: FC<RootProps & ConnectedProps & DispatchProps> = ({
   isFetchingSSOSettings,
   doGetSSOSettings,
 }) => {
+  useEffect(() => {
+    const style = document.createElement('style');
+    style.textContent = `.sr-only-focus:focus, .sr-only-focus:focus-visible { outline: none !important; }`;
```

Comment:
Is it possible to write a CSS class and apply it instead of manipulating the style directly?

Reply:
as discussed added inline comment